pilot/docker/Dockerfile.proxy_debug

```diff
@@ -1,40 +1,15 @@
-FROM ubuntu:xenial
-# Do not add more stuff to this list that isn't small or critically useful.
-# If you occasionally need something on the container do
-# sudo apt-get update && apt-get whichever
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
-      curl \
-      iptables \
-      iproute2 \
-      iputils-ping \
-      dnsutils \
-      netcat \
-      tcpdump \
-      net-tools \
-      libc6-dbg gdb \
-      elvis-tiny \
-      lsof \
-      busybox \
-      sudo && \
-    rm -rf /var/lib/apt/lists/*
+FROM istionightly/base_debug
```

Comment:
let's not base our releases on your org and unknown content
please 
- commit the dockerfile for that base
- make that base image exist in istio/ first

Reply:
Istionightly is the repo configured for the CircleCI - we want to built it nightly (and automatically).
I don't want CircleCI to have permissions for istio/ org.

For release: if the image is built first it'll use the local version instead of downloading.

Reply:
will the release process build it by default (how/without change? or which change)? why don't we use the (official) dailies or the latest version and rebuild it as part of the actual release process

cc @hklai

Reply:
Release script needs to call 'docker.basedebug' first. Will check with the release team to make sure this happens.

Reply:
I think we should treat the release as an (important) api and not keep making changes that require downstream processes to change to adapt to circleci or some dev specific workflows - those should go into special targets

Reply:
Release is an important step, and I agree we should test it and including in the build process.

I don't mind someone adding a 'make release' target, and moving the gcloud-specific commands to a separate upload step.

Reply:
any existing target you know the release already calls that you can add `docker.basedebug` to ? @chxchx do you know ?